src/observar.ts

```diff
@@ -0,0 +1,66 @@
+export const popovers = new Set<HTMLElement>();
+
+const handleChildListMutation = (mutation: MutationRecord) => {
+  if (mutation.addedNodes.length > 0) {
+    mutation.addedNodes.forEach((node) => {
+      if (node instanceof HTMLElement) {
+        if (node.hasAttribute('popover')) {
```

Comment:
I think you might also need to add `quetySelectorAll('[popover]')` here too. Might be worth adding a test for nodes that get added that have a popover element as a child.

Reply:
good catch!

Reply:
done